### ISSUE TITLE
WCAG 2.1 AA

### ISSUE THREAD

**Issue by mgifford** [SSAgov/ANDI#61]
Any plans to test for WCAG 2.1 criterion?

**Comment by JohnCotterSSA** [comment]
Hi, ANDI already tests for some WCAG 2.1 criterion. By design, ANDI doesn't specifically call out or reference WCAG failures or successes. If you come across something specific in WCAG 2.1 that you think ANDI could help with, please let us know.
